frontend/packages/dev-console/src/components/pipelines/PipelineVisualizationTask.scss

```
@@ -46,15 +57,15 @@ $gutter: 1.8em;
   }
 
   &__status {
-    margin: 0 4px;
+    margin-left: 7px;
```

I think we should throw all the margin in this basket as it's spacing purely for the icon...
```suggestion
    margin: 0 3px 0 7px;
```